SU2_CFD/include/variables/CDiscAdjFEABoundVariable.hpp

```diff
@@ -46,69 +47,93 @@
  * \author R. Sanchez.
  * \version 6.2.0 "Falcon"
  */
-class CDiscAdjFEABoundVariable : public CDiscAdjFEAVariable {
-protected:
+class CDiscAdjFEABoundVariable final : public CDiscAdjFEAVariable {
+private:
 
-  su2double *FlowTraction_Sens;         /*!< \brief Adjoint of the flow tractions. */
-  su2double *SourceTerm_DispAdjoint;    /*!< \brief Source term applied into the displacement adjoint
-                                                    coming from external solvers. */
+  MatrixType FlowTraction_Sens;        /*!< \brief Adjoint of the flow tractions. */
+  MatrixType SourceTerm_DispAdjoint;   /*!< \brief Source term applied into the displacement
+                                                   adjoint coming from external solvers. */
 
-public:
-
-  /*!
-   * \brief Constructor of the class.
-   */
-  CDiscAdjFEABoundVariable(void);
+  CVertexMap<unsigned> VertexMap;  /*!< \brief Object that controls accesses to the variables of this class. */
```

Comment:
Is it possible we can use this mapping object in other places? For example, isolating all boundary nodes quickly when preparing output.

Reply:
Hummm probably not as the mapping is in the opposite direction ( ivertex = f(ipoint) )... this is more of a workaround for the structural solvers not having "vertices"